src/librustc/hir/lowering.rs

```diff
@@ -1852,7 +1852,7 @@ impl<'a> LoweringContext<'a> {
                 // Record the introduction of 'a in `for<'a> ...`.
                 if let hir::GenericParamKind::Lifetime { .. } = param.kind {
                     // Introduce lifetimes one at a time so that we can handle
-                    // cases like `fn foo<'d>() -> impl for<'a, 'b: 'a, 'c: 'b + 'd>`.
+                    // cases like `fn foo<'d>() -> impl for<'a, 'b, 'c>`.
```

Comment:
Was this intentional?

Reply:
Comments were supposed to not be touched so this is indeed an oversight, oops.